product-service/.env

```diff
@@ -0,0 +1,5 @@
+PG_HOST=host
+PG_PORT=port
```

Comment:
you can add it to git ignore

Reply:
Thanks for the comment.  This is done intentionally.
The idea is that once the developer pulls this repo, he can then copy-paste this file and create another env file with a local suffix (i.e. `.env.local` which is already added to gitignore). In that file now it's safe to change env variables to real ones.